src/Blazored.Toast/BlazoredToasts.razor.cs

```diff
@@ -78,16 +78,16 @@ private ToastSettings BuildToastSettings(ToastLevel level, RenderFragment messag
             switch (level)
             {
                 case ToastLevel.Error:
-                    return new ToastSettings(string.IsNullOrWhiteSpace(heading) ? "Error" : heading, message, IconType, "blazored-toast-error", ErrorClass, ErrorIcon, ShowProgressBar);
+                    return new ToastSettings(heading, message, IconType, "blazored-toast-error", ErrorClass, ErrorIcon, ShowProgressBar);
 
                 case ToastLevel.Info:
-                    return new ToastSettings(string.IsNullOrWhiteSpace(heading) ? "Info" : heading, message, IconType, "blazored-toast-info", InfoClass, InfoIcon, ShowProgressBar);
+                    return new ToastSettings( heading, message, IconType, "blazored-toast-info", InfoClass, InfoIcon, ShowProgressBar);
 
                 case ToastLevel.Success:
-                    return new ToastSettings(string.IsNullOrWhiteSpace(heading) ? "Success" : heading, message, IconType, "blazored-toast-success", SuccessClass, SuccessIcon, ShowProgressBar);
+                    return new ToastSettings(heading, message, IconType, "blazored-toast-success", SuccessClass, SuccessIcon, ShowProgressBar);
 
                 case ToastLevel.Warning:
-                    return new ToastSettings(string.IsNullOrWhiteSpace(heading) ? "Warning" : heading, message, IconType, "blazored-toast-warning", WarningClass, WarningIcon, ShowProgressBar);
+                    return new ToastSettings(heading, message, IconType, "blazored-toast-warning", WarningClass, WarningIcon, ShowProgressBar);
```

Comment:
Unfortunately, we can't do this. This change would stop headings showing for everyone who is currently relying on the default behaviour, which is the majority of users.

You will need to update `IToastService` and `ToastService` and add the default heading values in there, instead of the current empty string values. For example:

```csharp
public void ShowInfo(string message, string heading = "")
```

Would become

```csharp
public void ShowInfo(string message, string heading = "Info")
```